ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -222,6 +222,12 @@
     reason="Test run only on LSO deployed cluster",
 )
 
+skipif_rhel_os = pytest.mark.skipif(
+    (config.ENV_DATA.get("rhel_workers", None) is True)
+    or (config.ENV_DATA.get("rhel_memory", None) is not None),
```

Comment:
I see that here https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_worker_nodes_network_failures.py#L87 we use the condition `is_rhel = config.ENV_DATA.get("rhel_workers") or config.ENV_DATA.get(
                    "rhel_user"
                )` to determine if it's "rhel" or not